Dockerfile.prod

```diff
@@ -0,0 +1,34 @@
+FROM python:3.10
+
+ENV PYTHONUNBUFFERED 1
+ARG REQUEST_BROKER_DNS
+ARG APPLICATION_PORT
+RUN apt-get update
+RUN apt-get install --yes apache2 apache2-dev 
+RUN apt-get install --yes postgresql
+RUN apt-get -y install python3-pip
+RUN pip install --upgrade pip
+
+RUN wget https://github.com/GrahamDumpleton/mod_wsgi/archive/refs/tags/4.9.0.tar.gz \
+    && tar xvfz 4.9.0.tar.gz \
+    && cd mod_wsgi-4.9.0 \
+    && ./configure --with-apxs=/usr/bin/apxs --with-python=/usr/local/bin/python \
```

Comment:
I'm presuming you've played with the flags here and have worked out the best way to configure. I'm just wondering if the `--with-python` flag is necessary given that this is a Python image so one would assume that the correct version of Python is on the system path already?